documentation/sphinx/source/mr-status-auto-gen.rst.inc

```diff
@@ -0,0 +1,620 @@
+.. -*- mode: rst; -*-
+
+.. |json-status-format| replace::
+   "cluster":{
+      "layers":{
+         "_valid":true,
+         "_error":"some error description"
+      },
+      "processes":{
+         "$map":{
```

Comment:
I think we use two special keywords in our schema. One is `$enum`, which may be straightforward enough to leave as is. The other is `$map`, which I feel more uncertain about. In particular, this schema doesn't provide any information about what the keys represent (other than it implicitly being a string type, I think), but this feels like a substantive loss.

Reply:
So I remove the $map {} in the generated document, will it be more acceptable?

Reply:
I think we need something that describes what's in the map. For example, I think the machines map has keys that are IP addresses, and that would ideally be apparent from the documentation.

Reply:
I see what you meant now. 
How about replacing $map in Schemas.cpp to $map_key=<name_of_the_key>?
There are 7 occurrences of $map in Schemas.cpp, we can manually encode them in a separate document and let the script to do the auto-replacement. 
If any $map is missing in the future, the script reports error. 

What do you think?